# Experiment 88b5fdbb480f353df55e33859231cedc346b43649b62c6f26654b2edf404b84b
# Base 07ff7cbd8bef0dfa5407c1637452c95f7b41f2f2
diff --git a/src/stat_obj.cpp b/src/stat_obj.cpp
index 9b05434f4..e650964da 100644
--- a/src/stat_obj.cpp
+++ b/src/stat_obj.cpp
@@ -628,8 +628,8 @@ void RGE_Static_Object::draw_front_frame(TDrawArea* param_1, short param_2, shor
     if (ot > 3) return;
     switch (ot) {
     case 0: this->draw_frame(param_1, param_2, param_3); return;
-    case 2: case 3: this->draw_frame_3d_cube_front(param_1,param_2,param_3); return;
-    case 1: this->draw_frame_3d_square_front(param_1,param_2,param_3); return;
+    case 1: this->draw_frame_3d_cube_front(param_1,param_2,param_3); return;
+    case 2: case 3: this->draw_frame_3d_square_front(param_1,param_2,param_3); return;
     }
 }
 
